docs/pages/application-access/guides/api-access.mdx

```diff
@@ -0,0 +1,133 @@
+[Postman](https://www.postman.com/).
+
+## Prerequisites
+
+We will assume that you followed the [Getting Started](../getting-started.mdx)
+guide or the general [App Access Usage](./usage.mdx) guide to connect the web
+application providing an API to Teleport.
+
+For the simplicity of the guide, we will be using Grafana running in a
+[Docker container](https://grafana.com/docs/grafana/latest/installation/docker/)
+and run API queries against it. You can launch Grafana too with a single Docker
+command:
+
+```sh
+$ docker run -d -p 3000:3000 grafana/grafana
+```
+
+Connect Grafana to your Teleport cluster by adding the following section in
+the Teleport App Service configuration file:
+
+```yaml
+app_service:
+  enabled: yes
+  apps:
+  - name: "grafana"
+    description: "Test Grafana server"
+    uri: "http://localhost:3000"
+```
+
+## Accessing API
+
+Log into your Teleport cluster and view available applications:
+
+```sh
+$ tsh login --proxy=teleport.example.com:3080
+$ tsh app ls
+Application Description         Public Address               Labels
+----------- ------------------- ---------------------------- ------
+grafana     Test Grafana server grafana.teleport.example.com
+```
+
+Retrieve short-lived X.509 certificate for the application:
+
+```sh
+$ tsh app login grafana
+Logged into app grafana. Example curl command:
+
+curl \
+  --cacert /Users/alice/.tsh/keys/teleport.example.com/certs.pem \
+  --cert /Users/alice/.tsh/keys/teleport.example.com/alice-app/cluster-name/grafana-x509.pem \
+  --key /Users/alice/.tsh/keys/teleport.example.com/alice \
+  https://grafana.teleport.example.com:3080
+```
+
+The login message shows an example `curl` command you can run to call the
+target application's API through Teleport App Access. Note the paths to the
+CA certificate and your user's certificate/key pair in the command - `curl`
+will use a client certificate to authenticate with Teleport.
+
+As Grafana's API requires authentication, let's update the `curl` command to
+provide basic auth information using default Grafana username/password and
+call its `/api/users` endpoint:
+
+```sh
+$ curl --user admin:admin \
+    --cacert /Users/alice/.tsh/keys/teleport.example.com/certs.pem \
+    --cert /Users/alice/.tsh/keys/teleport.example.com/alice-app/cluster-name/grafana-x509.pem \
+    --key /Users/alice/.tsh/keys/teleport.example.com/alice \
+    https://grafana.teleport.example.com:3080/api/users
+[{"id":1,"name":"","login":"admin","email":"admin@localhost","avatarUrl":"/avatar/46d229b033af06a191ff2267bca9ae56","isAdmin":true,"isDisabled":false,"lastSeenAt":"2021-03-18T17:25:59Z","lastSeenAtAge":"\u003c 1m","authLabels":[]}]
+```
+
+The app's X.509 certificate will expire on its own after the TTL allowed by
+your user's role. You can also remove it explicitly:
+
+```sh
+$ tsh app logout
+Logged out of app "grafana"
+```
+
+## App Information
+
+Teleport provides a way to see information about the application you're
```

Comment:
I would try to simplify this sentence to something like: `tsh app config` shows current app URI and paths to the secrets